src/main/java/seedu/duke/Parser.java

```diff
@@ -178,7 +179,8 @@ public static Command checkAction(String inputCommand) throws DukeExceptions {
             break;
         case "done":
             if (isHomeView) {
```

Comment:
do you wanna change it to if-else? Just a suggestion for better readability but this works too!

Reply:
@riazaham the else block would be blank though right?